test/install_test.go

```diff
@@ -49,12 +49,14 @@ var (
 		"linkerd-proxy-injector": {1, []string{"proxy-injector"}},
 	}
 
+	injectorErr = `.* linkerd-proxy-injector-.*-.* proxy-injector time=".*" level=warning msg="failed to retrieve replicaset from indexer, retrying with get request .*-%s/smoke-test-.*-.*: replicaset\.apps \\"smoke-test-.*-.*\\" not found"`
+
 	// Linkerd commonly logs these errors during testing, remove these once
 	// they're addressed: https://github.com/linkerd/linkerd2/issues/2453
 	knownControllerErrorsRegex = regexp.MustCompile(strings.Join([]string{
 		`.* linkerd-controller-.*-.* tap time=".*" level=error msg="\[.*\] encountered an error: rpc error: code = Canceled desc = context canceled"`,
 		`.* linkerd-web-.*-.* web time=".*" level=error msg="Post http://linkerd-controller-api\..*\.svc\.cluster\.local:8085/api/v1/Version: context canceled"`,
-		`.* linkerd-proxy-injector-.*-.* proxy-injector time=".*" level=warning msg="failed to retrieve replicaset from indexer, retrying with get request .*-smoke-test/smoke-test-.*-.*: replicaset\.apps \\"smoke-test-.*-.*\\" not found"`,
+		`.* linkerd-proxy-injector-.*-.* proxy-injector time=".*" level=warning msg="failed to retrieve replicaset from indexer, retrying with get request .*-smoke-test-.*-.*: replicaset\.apps \\"smoke-test-.*-.*\\" not found"`,
```

Comment:
i think this should be `.*-smoke-test.*/smoke-test-.*-.*`

Reply:
Indeed! :-)